system/edit-config.in

```diff
@@ -4,7 +4,12 @@
 
 file="${1}"
 
-EDITOR="${EDITOR-vi}"
+if [ $(command -v editor) ] ; then
```

Comment:
I think this will produce `stderr` log and possibly syntax error when the result is empty.

Reply:
Works just fine in at least ZSH and bash with no errors, though it is missing quotes that I meant to be there.  Unless I'm misreading the POSIX spec, it should work fine in any POSIX compliant shell once those quotes are added.